connect/connect.go

```diff
@@ -0,0 +1,61 @@
+// Package connect provides a Connector which allows to connect method
+// definition with their implementations in plugins
+package connect
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/nextmv-io/sdk/plugin"
+)
+
+// Connector connects methods with their implementations in plugins.
+type Connector struct {
```

Comment:
```suggestion
type connector struct {
```
Should this be unexported? I don't see it being actively referenced as `connect.Connector` anywhere outside this package, right?

Reply:
It is used in all packages that use the Connector, so we can't unexport. We could introduce an interface instead, but I don't see value in that at this time.

Reply:
Yeah but you're using `Connect` and `NewConnector`, right? You're not referencing the `Connector` type directly, that's why I suggested having the type be private, whilst of course keeping the functions public.

Reply:
The linter doesn't like that because the returned type would be unexported.